scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml

```diff
@@ -0,0 +1,194 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: sbd
```

Comment:
Actually... description says it should use IPMI, so there should not be a need for sbd (not even the nodes below, I guess)